### PR TITLE
Auth0 Dashboards

### PR DESCRIPTION
**Pull request by xer0x**
# PR Details

Add dashboards for Auth0

## Description

This adds a new dashboard named "Auth0 - Dashboard". It also has the queries and dashboards from the current Auth0-App.

## Checklist

- [x] Changes follows the best practices from the given [docs](https://docs.google.com/document/d/1xfYfruru0RFWOH23GRrRzosSlepJUTcakLFHLHdpEtc/edit?usp=sharing).
- [ ] Updated CHANGELOG.md.
- [x] Ran unit tests locally.
- [x] Updated screenshots/sample logs if any.

### CODE REVIEW

**Review by sharmoo — APPROVED** [review]

**Comment by sharmoo** [comment]
FYI, this is our Partner App Doc as well: https://docs.google.com/document/d/1iWz6Frd59HR3R0EIjBnXYQGZcJ8N87OYz5y6SolA5GI/edit?copiedFromTrash#

File: Auth0/Auth0.manifest.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "author": "Auth0",
+    "categories": [
+        "Compliance and Security",
+        "Sumo Logic Certified"
+    ],
+    "communityURL": "https://support.sumologic.com/hc/en-us/community/topics/200263058-Applications-and-Integrations",
+    "description": "Auth0 is a cloud-based, extensible identity provider for various applications. The Sumo Logic App for Auth0 makes it easy to analyze and visualize your Auth0 logs, and provides insight into security and operational issues.",
+    "helpURL": "https://auth0.com/docs/logs/streams/sumo-logic-dashboard",
+    "hoverText": null,
+    "iconURL": "https://app_icons.s3.amazonaws.com/auth0.png",
```
Comment (xer0x) [inline Auth0/Auth0.manifest.json:11]: @sumo-drew oops, was I supposed to leave the `iconURL` value alone?

**Comment by xer0x** [comment]
@sumo-drew we're ready to merge. Please take a look @sumo-apps-team

File: Auth0/Auth0.manifest.json (hunk 1 above)
Comment (sumo-drew) [reply Auth0/Auth0.manifest.json:11]: I don't think it matters as we set this during build and deploy. @himanshu219 can confirm.

Comment (sumo-apps-team) [reply Auth0/Auth0.manifest.json]: yes it doesn't matter.

Comment (sharmoo) [reply Auth0/Auth0.manifest.json]: Okay @sumo-apps-team - if we fix this really fast, is there still time to deploy by Monday?

Comment (sharmoo) [reply Auth0/Auth0.manifest.json]: @sumo-drew @sumo-apps-team Can I just delete these last two commits then and we can get this merged?

Comment (himanshu219) [reply Auth0/Auth0.manifest.json:11]: which two commits? actually when the test will run they automatically replace these urls and you will see a commit in your PR with description manifest updates.

**Review by sumo-apps-team — APPROVED** [review]
LGTM

**Pull request merged**